build_debian.sh

```diff
@@ -256,24 +256,19 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install apparmor
 sudo cp files/image_config/ntp/ntp-apparmor $FILESYSTEM_ROOT/etc/apparmor.d/local/usr.sbin.ntpd
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install apt-transport-https \
                                                        ca-certificates \
-                                                       curl \
-                                                       gnupg2 \
```

Comment:
>gnupg2 [](http://example.com/codeflow?start=55&length=6)

why remove gnupg2?

Reply:
It was being removed in line 275 below, after docker-ce was installed. This package is technically not needed for the purpose of installing docker-ce.